plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/util/ObjectStoreLocationProvider.java

```diff
@@ -32,7 +32,12 @@ public class ObjectStoreLocationProvider
         implements LocationProvider
 {
     private static final HashFunction HASH_FUNC = murmur3_32_fixed();
-    private static final Base64.Encoder BASE64_ENCODER = Base64.getUrlEncoder().withoutPadding();
+    // Length of entropy generated in the file location
```

Comment:
Can we also link `LocationProviders` code snippet https://github.com/apache/iceberg/blob/849f218d8d9a6783ba04d2d4eafc07754ec3885c/core/src/main/java/org/apache/iceberg/LocationProviders.java#L131-L136 here ?
These values may seem rather mystical to a naive reader of the `trino-iceberg` code.

Reply:
We already have a link here to the class. This is sufficient to me:

https://github.com/trinodb/trino/blob/400e08134b63fbc9d57e9f77b4cc5444546f4bdb/plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/util/ObjectStoreLocationProvider.java#L30